contracts/modules/common/ArgentSafeMath.sol

```diff
@@ -0,0 +1,42 @@
+/* The MIT License (MIT)
+
+Copyright (c) 2016 Smart Contract Solutions, Inc.
```

Comment:
2020, Argent Labs?

Reply:
Sorry, yes this was a leftover from the old `SafeMath` lib we had.